libs/database-entity/src/dto.rs

```diff
@@ -397,11 +408,19 @@ pub struct AFCollabMember {
   pub permission: AFPermission,
 }
 
-#[derive(Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize)]
 pub struct PublishInfo {
   pub namespace: Option<String>,
   pub publish_name: String,
   pub view_id: Uuid,
+  pub publisher_email: String,
```

Comment:
You should provide default value. Otherwise, new frontend client use old version appflowy-cloud will get serde error

Reply:
In what scenario will there be new frontend client using old appflowy-cloud? I'd assumed that we will be upgrading appflowy-cloud first before the releasing new appflowy flutter

Reply:
For self hosting user, mostly. For us there should be minimal impact.

Reply:
added default values